internal/rewrite_action.go

```diff
@@ -152,87 +152,48 @@ func (t *ImageTemplate) Render(chart *chart.Chart, rewriteActions ...*RewriteAct
 }
 
 func (t *ImageTemplate) Apply(originalImage name.Reference, rules *rewrite.Rules) ([]*RewriteAction, error) {
```

Comment:
@petewall I thought that this patch might be the right one to apply my refactoring on the apply function since I am changing it anyways. PTAL, happy to pair or discuss the changes